_js/main.go

```diff
@@ -14,7 +14,19 @@ import (
 )
 
 func main() {
-	exps := js.Module.Get("exports")
+	var exps *js.Object
+
+	if js.Module != js.Undefined {
+		exps = js.Module.Get("exports")
+	} else {
+		exps = js.MakeWrapper(map[string]interface{}{})
+		originalGlobalSh := js.Global.Get("sh")
+		exps.Set("noConflict", func() *js.Object {
+			js.Global.Set("sh", originalGlobalSh)
+			return exps
+		})
+		js.Global.Set("sh", exps)
+	}
```

Comment:
I also don't understand what any of this does, sorry - please document it.

Reply:
In this condition, `module` is not available, so we need to assign `window.sh` to get access to exported `sh` object outside.